examples/Benchmark/Performance/benchmark_cubes.pscn

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" ?>
+<?php echo '<!-- Generated from benchmark_cubes.pscn -->';?>
+
+<Node name="root" dt="0.01" gravity="0 -9.81 0">
+    <RequiredPlugin pluginName="SofaOpenglVisual"/>
+    <RequiredPlugin pluginName='SofaConstraint'/>
+    <RequiredPlugin pluginName='SofaMeshCollision'/>
+    <RequiredPlugin pluginName='SofaRigid'/>
+    <RequiredPlugin pluginName='SofaImplicitOdeSolver'/>
+    <RequiredPlugin pluginName='SofaLoader'/>
+    <RequiredPlugin pluginName='SofaGeneralEngine'/>
+
+
+    <VisualStyle displayFlags="showBehavior showCollisionModels" />
+    <OglSceneFrame/>
+
+    <!-- Basic Components to perform the collision detection -->
+    <FreeMotionAnimationLoop name="FreeMotionAnimationLoop" parallel="true" />
+    <DefaultPipeline name="CollisionPipeline" />
+    <BruteForceDetection name="BruteForceDetection" />
+    <LocalMinDistance name="Proximity" alarmDistance="0.2" contactDistance="0.09" angleCone="0.0" />
```

Comment:
I don't understand how I should use this parameter. It is a factor, and yet the default value is 0.5. I expected it to be 1. In the case of the rigid cubes, what value do you suggest? More than 1?
If I understand it correctly, the goal is to open the angle of the cone, but not too much to avoid too many contacts. Right?